invokeai/app/invocations/flux_redux.py

```diff
@@ -61,18 +65,53 @@ class FluxReduxInvocation(BaseInvocation):
         title="FLUX Redux Model",
         ui_type=UIType.FluxReduxModel,
     )
+    downsampling_factor: int = InputField(
+        ge=1,
+        le=9,
+        default=1,
+        description="Redux Downsampling Factor (1-9)",
+    )
+    downsampling_function: DOWNSAMPLING_FUNCTIONS = InputField(
+        default="area",
+        description="Redux Downsampling Function",
+    )
+    weight: float = InputField(
+        ge=0,
+        le=1,
+        default=1.0,
+        description="Redux weight (0.0-1.0)",
+    )
 
     def invoke(self, context: InvocationContext) -> FluxReduxOutput:
         image = context.images.get_pil(self.image.image_name, "RGB")
 
         encoded_x = self._siglip_encode(context, image)
         redux_conditioning = self._flux_redux_encode(context, encoded_x)
+        if self.downsampling_factor > 1 and self.weight != 1.0:
```

Comment:
I believe this should be:
```suggestion
        if self.downsampling_factor > 1 or self.weight != 1.0:
```